waspc/src/Wasp/Generator/SdkGenerator/Auth/OAuthAuthG.hs

```diff
@@ -40,7 +40,7 @@ genHelpers auth =
     mkHelpersFd :: OAuthAuthProvider -> Path' Rel' File' -> FileDraft
     mkHelpersFd provider helpersFp =
       mkTmplFdWithDstAndData
-        [relfile|auth/helpers/Generic.tsx|]
+        [relfile|auth/helpers/_Provider.tsx|]
```

Comment:
@infomiho This is the convention the rest of our code uses for template files that change their name and output multiple source files. Hope it's ok!

Reply:
Great change 👍